unicodetools/data/emoji/13.1/emoji-zwj-sequences.txt

```diff
@@ -0,0 +1,1183 @@
+# emoji-zwj-sequences.txt
```

Comment:
This does not appear to have the 13.1 zwj sequences yet; is it supposed to? (e.g. U+1F62E U+200D U+1F4A8)